setuptools/independent/RuntimeGraphGenerator/RuntimeGraphManager.cs

```diff
@@ -24,8 +24,8 @@ public NuGet.RuntimeModel.RuntimeGraph Collect(IEnumerable<LibraryExport> export
                     if (runtimeJson != null)
                     {
                         // Convert the package-name to lower-case in the path to lookup runtime.json
-                        string lowercasedPackageName = description.Identity.Name.ToLower();
-                        string pathToPackage = export.Library.Path.Replace(description.Identity.Name, lowercasedPackageName);
+                        // string lowercasedPackageName = description.Identity.Name.ToLower();
```

Comment:
Why comment this out?  I'm not sure I understand why we would need to change this.

Also, if this is correct, we need to update the comment above.